app/utils/log-context.js

```diff
@@ -11,11 +11,18 @@ function logContextMiddleware (req, res, next) {
   })
 }
 
+function addLoggingField (key, value) {
```

Comment:
Not sure what we'll really use this for in this task, but don't see anything wrong with what's been done.